### DISCUSSION TITLE
Better discoverability of connection deletion functionality

### DISCUSSION THREAD

**Discussion by rowanseymour** [nyaruka/floweditor#1131] [Ideas]
Hope this isn't just me misunderstanding the fundamentals of flow building, but say I want to break the connection from Other to the Send Message node below

<img width="304" alt="Captura de Pantalla 2019-10-25 a la(s) 10 00 48" src="https://user-images.githubusercontent.com/675558/67581914-6fdf0380-f70e-11e9-8ff1-c24f77be9a44.png">

The only way I know to do this is to drag the arrow away from the Send Message node, create another node like another Send Message action, OK, then delete that node.

**Comment by nicpottier** [comment]
You can click the half circle below Other, but ya, should maybe hover to offer a hint?

**Comment by rowanseymour** [comment]
Well darn... what else don't I know? Yeah maybe an grey X appears on hover, becomes red when clicked
